lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_workspace_runtime.rb

```diff
@@ -53,7 +53,8 @@ def self.instantiate_with_user(uri, user, attrs)
       self.current = workspace
       workspace.instantiate(uri, user, nil)
       workspace
-    rescue MiqAeException
+    rescue MiqAeException => err
+      $miq_ae_logger.info(err.message)
```

Comment:
This should log to `error` level instead of `info`.

Reply:
Fixed